data/ui/application.ui

```diff
@@ -141,6 +141,27 @@
             <property name="position">1</property>
           </packing>
         </child>
+        <child>
+          <object class="GtkButton" id="category_filter_button">
+            <property name="visible">True</property>
+            <property name="can_focus">True</property>
+            <property name="receives_default">False</property>
+            <property name="tooltip_text" translatable="yes" context="category_filter" comments="Tooltip for category filter button">Category filters</property>
+            <property name="valign">center</property>
+            <signal name="clicked" handler="on_menu_category_filter_clicked" swapped="no"/>
+            <child>
+              <object class="GtkImage">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="icon_name">filter-symbolic</property>
```

Comment:
This icon is not included into Ubuntu 22.04 and Debian 12 by default, so we'll need to use another icon.

Reply:
Okay, I'll change it. Is there a list somewhere of available icons?

Reply:
I've replaced the icon with `view-conceal-symbolic` which I found in the `/usr/share/icons/Adwaita` folder (Adwaita being a Gnome theme, so hopefully that icon should be included in stock Ubuntu and Debian). Let me know if that one works.